plugins/github/github.js

```diff
@@ -158,9 +158,15 @@ exports.Github = (function() {
       }
 
       if (isFinite(req.body.page) && +req.body.page > 1) {
-        uri += "&perpage=" +  githubConfig.repos_per_page + "&page=" + req.body.page
+        uri += "&page=" + req.body.page
       }
 
+      if (isFinite(req.body.per_page) && +req.body.per_page > 1) {
+        uri += "&per_page=" + req.body.per_page
+      }
+
+      uri += "&type=owner"
```

Comment:
This will make sure that we only get repos under the "organization" of the user. So if a the user of dillinger has the username "pam" it will only fetch repos like "pam/coolproject".

Effectively excluding repos where the user is a contributor and not a "full" owner.

We might wanna change this back to "all" and render the username before the repo name in the UI.